### DISCUSSION TITLE
Log-in problem (not caused by LXQt)

### DISCUSSION THREAD

**Discussion by tsujan** [lxqt/lxqt#2229] [Q&A]
I'm sure this isn't an issue in LXQt, but it affects LXQt; hence opening a discussion here.

After recent updates in Manjaro, `systemd` (which wasn't among the updated packages) started a new annoying behavior by giving this at logins randomly:

```
Apr 30 09:53:22 manjaro systemd[111174]: user@1000.service: Failed to attach to cgroup /user.slice/user-1000.slice/user@1000.service: Device or resource busy
Apr 30 09:53:22 manjaro systemd[111174]: user@1000.service: Failed at step CGROUP spawning /usr/lib/systemd/systemd: Device or resource busy
Apr 30 09:53:22 manjaro systemd[1]: user@1000.service: Failed with result 'exit-code'.
Apr 30 09:53:22 manjaro systemd[1]: Failed to start User Manager for UID 1000
```

The result was a lame session without user daemons and dbus. Logging out was also impossible by using the main menu; I had to do it in QTerminal (which wasn't among the affected apps):

```
session=`loginctl session-status | head -n 1 | awk '{print $1}'`
loginctl terminate-session $session
```

I may have seen a similar problem when logging into KDE. The difference was that, in the case of KDE, I encountered a blank screen, in which nothing worked, except for the power button.

**Comment by stefonarch** [comment]
In arch no problems atm - not quite helpful I know.

**Reply by tsujan** [reply]
I'll test the strange possibility that It may be related to NetworkManager and setting connection priorities. It happened after I set priorities for two wireless connections  — something I'd never done before.

But even if it's related to NetworkManager, the bug is in `systemd`.

**Reply by tsujan** [reply]
This was definitely related to NetworkManager's connection priorities. After I disabled priorities, I did tens of log-outs and log-ins (to test the new behavior of Qt6's `QCoreApplication::quit()` with GUI apps) but didn't encounter any problem, whether in LXQt or in KDE. And there was no update in the meanwhile.
